binding_test.go

```diff
@@ -346,6 +346,93 @@ func TestJSON(t *testing.T) {
 	})
 }
 
+func TestYaml(t *testing.T) {
+	t.Run("pointer model", func(t *testing.T) {
+		assert.PanicsWithValue(t,
+			"binding: pointer can not be accepted as binding model",
+			func() {
+				type yaml struct {
+					Username string
+					Password string
+				}
+				Yaml(&yaml{})
+			},
+		)
+	})
+
```

Comment:
Looks like a test similar to ["invalid JSON"](https://github.com/flamego/binding/blob/dbd21dca85f5a72c9ffe7fcfac551f5a40f93ac4/binding_test.go?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L36) is missing?

Reply:
@Germiniku I think here we are still missing the test case without "custom error handler".